src/DGmethods/DGBalanceLawDiscretizations.jl

```diff
@@ -166,14 +194,18 @@ stored in the auxiliary state.
 
 !!! todo
 
-    - Add support for boundary conditions
     - support viscous fluxes (`gradstates` is in the argument list as part of
       this future interface)
+    - Revisit how to handle plus side state in
+      `inviscid_numerical_boundary_flux!` after gradient state is handled (how
+      or not propagate changes without adverse side effects).
```

Comment:
"or not propagate" -> "to or not propagate"?

Reply:
done